packages/core/data-transfer/lib/providers/local-file-destination-provider.ts

```diff
@@ -90,6 +97,17 @@ class LocalFileDestinationProvider implements IDestinationProvider {
     fs.mkdirSync(path.join(rootDir, 'configuration'));
   }
 
+  close(): void | Promise<void> {
```

Comment:
Will this ever return a promise?

Reply:
No indeed, just auto-generated types from the parent class, will remove it